spec.md

```diff
@@ -126,6 +127,58 @@ response if the authentication fails.
 Note: Using an authentication mechanism that is agreed to via out of band
 communications could lead to interoperability issues with other platforms.
 
+## Originating Identity
+
+Often a service broker will need to know the identity of the user that
+initiated the request from the platform. For example, this might be needed
+for auditing or authorization purposes. In order to facilitate this, the
+platform will need to provide this identification information to the broker
+on each request. Platforms MAY support this feature, and if they do, they
+MUST adhere to the following:
+- for any OSB API request that is the result of an action taken by
```

Comment:
Caps?

Reply:
@angarg12 "Caps?" on which word(s) ?

Reply:
These 3 bullet points begin with a lowercase letter, - for, - any, - if

Reply:
ah ok, done.